client/server/ClassVisitor.js

```diff
@@ -22,37 +22,62 @@ class ClassVisitor {
         else if (node instanceof YmlToBdlParser_1.MemberDeclarationContext) {
             this.visitMemberDeclaration(node);
         }
+        else if (node instanceof YmlToBdlParser_1.MethodDeclarationContext) {
+            this.visitMethodDeclaration(node);
+        }
         else {
             this.visitChildren(node);
         }
     }
+    visitMethodDeclaration(node) {
+        this.createNewCompletionItem(node.methodIntro().ymlId(), node.field(), "Method", vscode_languageserver_1.CompletionItemKind.Method);
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/15327919/50592324-f383e700-0e93-11e9-97b0-f3cd1eaf201b.png)

Concerning this specific case, this is because we only check the declaration of the method, not its implementation. Note that `printValue()` is declared in the class `PrintableValuesObject`.
Also, don't forget that we get suggestions only after parsing an opened file since we don't use workspace feature for now.

![image](https://user-images.githubusercontent.com/15327919/50592628-3b573e00-0e95-11e9-8ccb-3dbf6875157b.png)

Reply:
Hum, so in the information popup there is nothing differentiating methods from text methods? I only see the return type.

Reply:
Nothing yet, sir.